lib/js/main.js

```diff
@@ -26,41 +25,51 @@ var sweep = (function() {
       var deltas = animation.deltas;
 
       // components = [h, s, l, a] || [r, g, b, a]
-      var composed, components = new Array(4);
+      var composed,
+        components = new Array(4);
 
       // set alpha level
       components[3] = from.a + deltas[3] * frame;
 
-      if (space === 'RGB') {
+      if (space === "RGB") {
         // red
         components[0] = Math.floor(from.r + deltas[0] * frame);
         // green
         components[1] = Math.floor(from.g + deltas[1] * frame);
         // blue
         components[2] = Math.floor(from.b + deltas[2] * frame);
 
-        composed = 'rgba(' + [components] + ')';
-
-      } else { // it's HSL(uv)
+        composed = "rgba(" + [components] + ")";
+      } else {
+        // it's HSL(uv)
         // hue
         components[0] = (Math.floor(from.h + deltas[0] * frame) + 360) % 360;
         // saturation
         components[1] = Math.floor((from.s + deltas[1] * frame) * 100);
         // lightness
         components[2] = Math.floor((from.l + deltas[2] * frame) * 100);
-        if (space === 'HSL') {
-          composed = 'hsla(' + components[0] + ',' +
-            components[1] + '%,' +
-            components[2] + '%,' +
-            components[3] + ')';
+        if (space === "HSL") {
+          composed =
+            "hsla(" +
+            components[0] +
+            "," +
+            components[1] +
+            "%," +
+            components[2] +
+            "%," +
+            components[3] +
+            ")";
         } else {
-          composed = 'rgba(' + hsluv.hsluvToRgb([
-            components[0],
-            components[1],
-            components[2]
-          ]).map(function(component) {
-            return Math.floor(component * 255);
-          }) + ',' + components[3] + ')';
+          composed =
```

Comment:
There's a lot of reformatting in this diff - reordered `require` lines, `'` => `"`, different line breaks... was this intentional?